.github/workflows/acceptance.yml

```diff
@@ -19,6 +19,10 @@ concurrency:
 env:
   SEGMENT_DOWNLOAD_TIMEOUT_MINS: 3
   NODE_OPTIONS: '--max-old-space-size=4096'
+  # Disables sentry reporting for devservices
+  DISABLE_SENTRY: 1
```

Comment:
hmm this environment variable seems not specific enough -- I would imagine `SENTRY_DEVSERVICES_...` prefix

Reply:
I'll change this to `DEVSERVICES_DISABLE_SENTRY` so it's a bit more fitting when used in non-sentry repos